modules/jumpbox/main.tf

```diff
@@ -131,6 +131,12 @@ resource "aws_instance" "jumpbox" {
     user = "ubuntu"
   }
 
+  lifecycle {
```

Comment:
What's the plan to update jumpboxes regularly enough to get upstream patches?

Reply:
We install unattended-upgrades on all hosts, so they pull in security updates automatically.